docs/pages/develop/user-interface/fonts.mdx

```diff
@@ -160,7 +199,9 @@ For example, to use Inter font you can install the [`@expo-google-fonts/inter`](
 
 <Terminal cmd={['$ npx expo install expo-font @expo-google-fonts/inter']} />
 
-Then, you can integrate it in your project by using the `useFonts` hook. You can directly use this hook from the Google Fonts package. Under the hood, the hook uses [`Font.loadAsync`](/versions/latest/sdk/font/#loadasyncfontfamilyorfontmap-source). You do not have to explicitly import the font file since that is done by the package itself.
+Then, you can integrate it in your project with the config plugin by passing in the path to the desired font file to the `fonts` option (eg: `node_modules/@expo-google-fonts/inter/Inter_100Thin.ttf`), or you can use the `useFonts` hook.
```

Comment:
````suggestion
Then, you can [embed it into your project with the config plugin](#embed-the-font-in-your-native-project) by passing the path to the desired font file to the `expo-font.fonts` option, or you can use the `useFonts` hook.

An example of the path provided to the `fonts` option in the config plugin:

```json app.json
{
  "plugins": [
    [
      "expo-font",
      {
        "fonts": "node_modules/@expo-google-fonts/inter/Inter_100Thin.ttf"
      }
    ]
  ]
}
```
````

- Let's add link to the config plugin section.
- Let's also use the same terminology (embed) instead
- Also, provide the example of provide for the `node_modules` path too